src/DeepCopy/Exception/DeepCopyExceptionInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace DeepCopy\Exception;
+
+interface DeepCopyExceptionInterface
+{
+}
```

Comment:
This interface was added but I don't think it's related to this pull request, could you remove it please?